sentry.gradle

```diff
@@ -68,18 +68,17 @@ gradle.projectsEvaluated {
 
           try {
             if (versionCode instanceof String) {
-                versionCode = Integer.parseInt(versionCode);
+                versionCode = Integer.parseInt(versionCode)
+                versionCode = Math.abs(versionCode)
```

Comment:
Abs? The version code might end up being parsed as a negative value? I.e. `-3.14`

Reply:
`abs` is already used, just moved the code.
`abs` make negative numbers into positive numbers, not the other way around.

Reply:
I got that. I was just surprised that the `versionCode` might end up with a negative value after being parsed from string.